### PR TITLE
Changed siprouting SDK to GA version.

### PR DESCRIPTION
**Pull request by jiriburant**
# Description

SIP routing GA version.

Regenerated SIP routing functionality in PhoneNumbers SDK to accomodate for GA version.

Rest API swagger: https://github.com/Azure/azure-rest-api-specs/pull/22747

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/cosmos/azure-spring-data-cosmos/README.md [hunk 1]
```diff
@@ -940,7 +940,7 @@ public class MultiDatabaseApplication implements CommandLineRunner {
 public class MultiTenantDBCosmosFactory extends CosmosFactory {
 
     private String tenantId;
-
```
Comment (danielortega-msft) [inline sdk/cosmos/azure-spring-data-cosmos/README.md:943]: Were these changes intended to be part of this PR?

**Review by danielortega-msft — CHANGES_REQUESTED** [review]
Please add a changelog entry stating that the API version for the SIP client changed.

File: sdk/communication/azure-communication-phonenumbers/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,8 @@
 ## 1.1.0-beta.15 (Unreleased)
 
 ### Features Added
+- API version `2022-12-01` is now the default for Phone Numbers clients.
```
Comment (danielortega-msft) [inline sdk/communication/azure-communication-phonenumbers/CHANGELOG.md:6]: This one is not necessary; it was already reflected in 1.1.0-beta.13 below.

File: sdk/cosmos/azure-spring-data-cosmos/README.md (hunk 1 above)
Comment (jiriburant) [reply sdk/cosmos/azure-spring-data-cosmos/README.md]: Reverted changes.

File: sdk/communication/azure-communication-phonenumbers/CHANGELOG.md (hunk 2 above)
Comment (jiriburant) [reply sdk/communication/azure-communication-phonenumbers/CHANGELOG.md]: Removed.

**Review by danielortega-msft — APPROVED** [review]
